### PR TITLE
Add Jest

### PR DESCRIPTION
**Pull request by terkelg**
Jest is a JavaScript testing framework, used by Facebook to test JavaScript code including React applications.
https://facebook.github.io/jest/

### CODE REVIEW

**Comment by sindresorhus** [comment]
Please read and adhere to the [contributing guidelines](../blob/master/contributing.md).

**Comment by terkelg** [comment]
Sorry, I missed a few points. I've updated the pull request.

File: readme.md [hunk 1]
```diff
@@ -606,7 +606,7 @@
 - [testen](https://github.com/egoist/testen) - Run tests for multiple versions of Node.js locally with NVM.
 - [Nightwatch](https://github.com/nightwatchjs/nightwatch) - Automated UI testing framework based on Selenium WebDriver.
 - [WebdriverIO](http://webdriver.io) - Automated testing based on the WebDriver protocol.
-
```
Comment (sindresorhus) [inline readme.md]: Don't remove the newline

File: readme.md [hunk 2]
```diff
@@ -606,7 +606,7 @@
...
+- [Jest](https://github.com/facebook/jest) - Painless JavaScript Testing.
```
Comment (sindresorhus) [inline readme.md]: Testing => testing

**Comment by terkelg** [comment]
Adjusted pull request to code review

**Pull request merged**